src/mmrelay/meshtastic/connection.py

```
@@ -1126,6 +1126,8 @@ def connect_iface(iface_param: Any) -> None:
                 )
                 _raise_no_client(connection_type)
 
+            if client is None:
+                raise RuntimeError("Unreachable: client validated above")
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)